### ISSUE TITLE
Scrollbar grip invisible until hovered over

### ISSUE THREAD

**Issue by codingthat** [ValveSoftware/steam-for-linux#6743]
This is frankly a bit difficult to use:

![Steam-scrollbar](https://user-images.githubusercontent.com/26723818/70356992-0bb76f80-1876-11ea-91a2-39623758d2c8.gif)

#### Your system information

* Steam client version (build number or date): Nov 29 2019 and Dec 6 2019
* Distribution (e.g. Ubuntu): Linux Mint 19.1
* Opted into Steam client beta?: [Yes/No] no
* Have you checked for system updates?: [Yes/No] yes (hence confirming this bug in two different builds as noted above)

#### Please describe your issue in as much detail as possible:

I expect that viewing discussions has a scroll bar whose grip is not invisible, but it is.

#### Steps for reproducing this issue:

1. Launch Steam
2. Store page scroll bar looks OK
3. Click mail icon
4. Click 0 new comments icon
5. Click Last chance for the Steam Autumn Sale & Steam Awards nominations!
6. Scroll bar is invisible (and wider than in step 2, when you do find it by hovering)

**Comment by arrowgent** [comment]
confirmed, this only occurs in certain areas of the steam website

**Comment by kisak-valve** [comment]
Hello @codingthat, @arrowgent, are either of you still experiencing this issue on an up to date system?

**Comment by codingthat** [comment]
Yes, still happening here!  Tested just now on Linux Mint 20.3 Una 64-bit.

![image](https://user-images.githubusercontent.com/26723818/167172753-ce3bf296-ab5d-4026-af1c-759f0a100fa1.png)

**Comment by codingthat** [comment]
(When I check for updates from within Steam, it says that's up-to-date, but just to be sure...)
